lib/src/core.dart

```diff
@@ -40,6 +40,12 @@ class AudioSession {
   AudioSessionConfiguration _configuration;
   final _configurationSubject = BehaviorSubject<AudioSessionConfiguration>();
   final _interruptionEventSubject = PublishSubject<AudioInterruptionEvent>();
+  final _routeChangeReasonSubject =
+      PublishSubject<AVAudioSessionRouteChangeReason>();
+  final _silenceSecondaryAudioHintSubject =
+      PublishSubject<AVAudioSessionSilenceSecondaryAudioHintType>();
+  final _mediaServicesWereLostSubject = PublishSubject<void>();
+  final _mediaServicesWereResetSubject = PublishSubject<void>();
```

Comment:
These can be removed, and an app can access `AVAudioSession` directly.

Reply:
Removed!